cli/cortx-prvsnr-cli.spec

```diff
@@ -20,6 +20,28 @@ CORTX Provisioner Command line interface. Provides utilities to deploy CORTX Obj
 %prep
 %setup -n %{name}-%{version}-%{_cortx_prvsnr_git_ver}
 
+/bin/cat <<EOL > ssh_config
+Host srvnode-1 srvnode-1.colo.seagate.com
```

Comment:
no, no, no. please DO NOT hard code colo.seagate.com here. our lab is not the only environment where the systems will operate

Reply:
@sradhanand-pati change this to:  
`Host srvnode-1 srvnode-1.localdomain`